pkg/minikube/mustload/mustload.go

```diff
@@ -111,10 +110,6 @@ func Running(name string) ClusterController {
 		exit.WithError("Unable to get driver IP", err)
 	}
 
-	if driver.IsKIC(host.DriverName) {
```

Comment:
If I am not wrong, I believe we still need this for macOS.
we can not hit the IP of the container but the DefaultBind 127.0.0.1